### ISSUE TITLE
[rancher-backup] skip tls quoting issue in Charts

### ISSUE THREAD

**Issue by mscbpi** [rancher/charts#896]
```  
insecureTLSSkipVerify: {{ .insecureTLSSkipVerify }}  
```

in `s3-secret.yaml`

has to be changed in
```  
insecureTLSSkipVerify: {{ quote .insecureTLSSkipVerify }}  
```

otherwise you end up with

`Error: Secret in version "v1" cannot be handled as a Secret: v1.Secret.StringData: ReadString: expects " or n, but found t, error found in #10 byte of ...|pVerify":true},"type|..., bigger context ...|t","folder":"k8s-backup","insecureTLSSkipVerify":true},"type":"Opaque"}
|...`

**Comment by mscbpi** [comment]
Oops this permits the helm deployment but it does not work
`cannot unmarshal string into Go struct field S3ObjectStore.insecureTLSSkipVerify of type bool` then in backup job...

**Comment by mrajashree** [comment]
@mscbpi this issue is now fixed in the chart template as well as the operator https://github.com/rancher/backup-restore-operator/issues/56

The fix will be available in 2.5.4 with the new version of this chart. This chart's s3-secret.yaml will quote the skip verify flag and has a new tag for backup-restore-operator v1.0.3 that will accept the value as a string

**Comment by mscbpi** [comment]
Ok in between I tried to deploy it without any default s3 and adding the destination in the job.

This skip value is never taken into account (it fails if unknown CA).

I tried to add my CA as documented, it won't work either, saying 
`open LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS.....: file name too long`
